wagtail/project_template/home/templates/home/welcome_page.html

```diff
@@ -0,0 +1,377 @@
+    -webkit-transform: scaleX(1.25);
+    transform: scaleX(1.25);
+    -webkit-transform-origin: center;
+    transform-origin: center;
+    stroke-width: 3px;
+  }
+  .box {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+  }
+  .circle {
+    fill: #6DDCBD;
+    stroke: #fff;
+    stroke-width: 7px;
+  }
+  h2 {
+    font-size: 22px;
+    max-width: 500px;
+    margin: 5px auto 0;
+  }
+  main p {
+    font-size: 16px;
+    line-height: 20px;
+    max-width: 390px;
+    margin: 15px auto 0;
+    color: #000000;
+  }
+  footer {
+    padding: 25px 0;
+    position: fixed;
+    left: 50%;
+    bottom: 0;
+    width: 960px;
+    -webkit-transform: translateX(-50%);
+    transform: translateX(-50%);
+    -webkit-transform-style: preserve-3d;
+    transform-style: preserve-3d;
+    border-top: 1px solid #efefef;
+  }
+  .option {
+    float: left;
+    width: 33.33%;
+    box-sizing: border-box;
+    padding-right: 5px;
+  }
+  .option svg {
+    width: 25px;
+    height: 25px;
+    fill: gray;
+    border: 1px solid #d6d6d6;
+    padding: 5px;
+    border-radius: 100%;
+    float: left;
+    margin-right: 10px;
+  }
+  .option div {
+    display: table;
+  }
+  .option h4 {
+    color: #246060;
+    font-size: 19px;
+  }
+  .option h4:hover {
+    text-decoration: underline;
+  }
+  .option p {
+    font-weight: 300;
+    font-size: 15px;
+    padding-top: 3px;
+    color: #000000;
+  }
+  @media (max-width: 996px) {
+    body, footer {
+      max-width: 780px;
+    }
+  }
+  @media (max-width: 800px) {
+    footer, main {
+      position: relative;
+    }
+    footer {
+      height: 100%;
+    }
+    .option {
+      position: relative;
+      width: 100%;
+      left: auto;
+      right: auto;
+      top: auto;
+      padding: 0 25px;
+      margin-bottom: 60px;
+    }
+    .two {
+      margin-left: 0px;
+      -webkit-transform: none;
+      transform: none;
+    }
+    .option {
+      float: none;
+    }
+    .figure {
+      margin-top: 10px;
+    }
+    main {
+      padding: 0 25px;
+    }
+    main h2 {
+      font-size: 18px;
+    }
+    main p {
+      font-size: 14px;
+    }
+    header {
+      padding-left: 20px;
+      padding-right: 20px;
+    }
+    footer {
+      overflow: hidden;
+      width: 100%;
+      margin-top: 50px;
+    }
+  }
+  @media (min-width: 801px) and (max-height: 730px) {
+    .figure {
+      margin-top: 80px;
+    }
+  }
+  @media (min-width: 801px) and (max-height: 600px) {
+    footer {
+      position: relative;
+      margin: 135px auto 0;
+    }
+    .figure {
+      margin-top: 50px;
+    }
+  }
+</style>
+<header>
+  <div class="row">
+    <div class="column">
+      <div class="logo">
+        <a href="https://wagtail.io/" target="_blank" rel="nofollower noopener">
+          <svg class="figure-logo" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 342.5 126.2">
+            <title>Visit the Wagtail website</title>
+            <g>
```

Comment:
Could you take this, and all other SVG on the page through [SVGO](https://jakearchibald.github.io/svgomg/)? There is a lot of unnecessary markup in here that makes it harder to see what is going on. Here is the output for this particular visual:

```svg
<svg class="figure-logo" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 342.5 126.2">
  <path fill="#FFF" d="M84 1.9v5.7s-10.2-3.8-16.8 3.1c-4.8 5-5.2 10.6-3 18.1 21.6 0 25 12.1 25 12.1L87 27l6.8-8.3c0-9.8-8.1-16.3-9.8-16.8z"/>
  <circle cx="85.9" cy="15.9" r="2.6"/>
  <path d="M89.2 40.9s-3.3-16.6-24.9-12.1c-2.2-7.5-1.8-13 3-18.1C73.8 3.8 84 7.6 84 7.6V1.9C80.4.3 77 0 73.2 0 59.3 0 51.6 10.4 48.3 17.4L9.2 89.3l11-2.1-20.2 39 14.1-2.5L24.9 93c30.6 0 69.8-11 64.3-52.1z"/>
  <path d="M102.4 27l-8.6-8.3L87 27z"/>
  <path fill="#FFF" d="M30 84.1s1-.2 2.8-.6c1.8-.4 4.3-1 7.3-1.8 1.5-.4 3.1-.9 4.8-1.5 1.7-.6 3.5-1.2 5.2-2 1.8-.7 3.6-1.6 5.4-2.6 1.8-1 3.5-2.1 5.1-3.4.4-.3.8-.6 1.2-1l1.2-1c.7-.7 1.5-1.4 2.2-2.2.7-.7 1.3-1.5 1.9-2.3l.9-1.2.4-.6.4-.6c.2-.4.5-.8.7-1.2.2-.4.4-.8.7-1.2l.3-.6.3-.6c.2-.4.4-.8.5-1.2l.9-2.4c.2-.8.5-1.6.7-2.3.2-.7.3-1.5.5-2.1.1-.7.2-1.3.3-2 .1-.6.2-1.2.2-1.7.1-.5.1-1 .2-1.5.1-1.8.1-2.8.1-2.8l1.6.1s-.1 1.1-.2 2.9c-.1.5-.1 1-.2 1.5-.1.6-.1 1.2-.3 1.8-.1.6-.3 1.3-.4 2-.2.7-.4 1.4-.6 2.2-.2.8-.5 1.5-.8 2.4-.3.8-.6 1.6-1 2.5l-.6 1.2-.3.6-.3.6c-.2.4-.5.8-.7 1.3-.3.4-.5.8-.8 1.2-.1.2-.3.4-.4.6l-.4.6-.9 1.2c-.7.8-1.3 1.6-2.1 2.3-.7.8-1.5 1.4-2.3 2.2l-1.2 1c-.4.3-.8.6-1.3.9-1.7 1.2-3.5 2.3-5.3 3.3-1.8.9-3.7 1.8-5.5 2.5-1.8.7-3.6 1.3-5.3 1.8-1.7.5-3.3 1-4.9 1.3-3 .7-5.6 1.3-7.4 1.6-1.6.6-2.6.8-2.6.8z"/>
  <g fill="#231F20">
    <path d="M127 83.9h-8.8l-12.6-36.4h7.9l9 27.5 9-27.5h7.9l9 27.5 9-27.5h7.9L153 83.9h-8.8L135.6 59 127 83.9zM200.1 83.9h-7V79c-3 3.6-7 5.4-12.1 5.4-3.8 0-6.9-1.1-9.4-3.2s-3.7-5-3.7-8.6c0-3.6 1.3-6.3 4-8 2.6-1.8 6.2-2.7 10.7-2.7h9.9v-1.4c0-4.8-2.7-7.3-8.1-7.3-3.4 0-6.9 1.2-10.5 3.7l-3.4-4.8c4.4-3.5 9.4-5.3 15.1-5.3 4.3 0 7.8 1.1 10.5 3.2 2.7 2.2 4.1 5.6 4.1 10.2v23.7zm-7.7-13.6v-3.1h-8.6c-5.5 0-8.3 1.7-8.3 5.2 0 1.8.7 3.1 2.1 4.1 1.4.9 3.3 1.4 5.7 1.4 2.4 0 4.6-.7 6.4-2.1 1.8-1.3 2.7-3.1 2.7-5.5zM241.7 47.5v31.7c0 6.4-1.7 11.3-5.2 14.5-3.5 3.2-8 4.8-13.4 4.8-5.5 0-10.4-1.7-14.8-5.1l3.6-5.8c3.6 2.7 7.1 4 10.8 4 3.6 0 6.5-.9 8.6-2.8 2.1-1.9 3.2-4.9 3.2-9v-4.7c-1.1 2.1-2.8 3.9-4.9 5.1-2.1 1.3-4.5 1.9-7.1 1.9-4.8 0-8.8-1.7-11.9-5.1-3.1-3.4-4.7-7.6-4.7-12.6s1.6-9.2 4.7-12.6c3.1-3.4 7.1-5.1 11.9-5.1 4.8 0 8.7 2 11.7 6v-5.4h7.5zm-28.4 16.8c0 3 .9 5.6 2.8 7.7 1.8 2.2 4.3 3.2 7.5 3.2 3.1 0 5.7-1 7.6-3.1 1.9-2.1 2.9-4.7 2.9-7.8 0-3.1-1-5.8-2.9-7.9-2-2.2-4.5-3.2-7.6-3.2-3.1 0-5.6 1.1-7.4 3.4-2 2.1-2.9 4.7-2.9 7.7zM260.9 53.6v18.5c0 1.7.5 3.1 1.4 4.1.9 1 2.2 1.5 3.8 1.5 1.6 0 3.2-.8 4.7-2.4l3.1 5.4c-2.7 2.4-5.7 3.6-8.9 3.6-3.3 0-6-1.1-8.3-3.4-2.3-2.3-3.5-5.3-3.5-9.1V53.6h-4.6v-6.2h4.6V36.1h7.7v11.4h9.6v6.2h-9.6zM309.5 83.9h-7V79c-3 3.6-7 5.4-12.1 5.4-3.8 0-6.9-1.1-9.4-3.2s-3.7-5-3.7-8.6c0-3.6 1.3-6.3 4-8 2.6-1.8 6.2-2.7 10.7-2.7h9.9v-1.4c0-4.8-2.7-7.3-8.1-7.3-3.4 0-6.9 1.2-10.5 3.7l-3.4-4.8c4.4-3.5 9.4-5.3 15.1-5.3 4.3 0 7.8 1.1 10.5 3.2 2.7 2.2 4.1 5.6 4.1 10.2v23.7zm-7.7-13.6v-3.1h-8.6c-5.5 0-8.3 1.7-8.3 5.2 0 1.8.7 3.1 2.1 4.1 1.4.9 3.3 1.4 5.7 1.4 2.4 0 4.6-.7 6.4-2.1 1.8-1.3 2.7-3.1 2.7-5.5zM319.3 40.2c-1-1-1.4-2.1-1.4-3.4 0-1.3.5-2.5 1.4-3.4 1-1 2.1-1.4 3.4-1.4 1.3 0 2.5.5 3.4 1.4 1 1 1.4 2.1 1.4 3.4 0 1.3-.5 2.5-1.4 3.4s-2.1 1.4-3.4 1.4c-1.3.1-2.4-.4-3.4-1.4zm7.2 43.7h-7.7V47.5h7.7v36.4zM342.5 83.9h-7.7V33.1h7.7v50.8z"/>
  </g>
</svg>
```

Reply:
Done in 959d41b.